### PR TITLE
#150834085/ #150840244 Add new learning map models and update seeding script

### PR DESCRIPTION
**Pull request by LarryWachira**
#### What does this PR do?
- Adds new learning map models: Skill, Resource, ResourceType, LearningUnit and LearningUnitType
- Adds two new association tables: path_level and path_level_skill
- Adds a management command to allow seeding data from LV dump CSVs to CALM's proposed tables.
- Removes label key in levels endpoint response

#### Description of Task completed:
- Create the following commands:
``` python
python manage.py seed_all

python manage.py seed_paths

python manage.py seed_test_data
```

- Refactor:
Path and Level models

#### How should this be manually tested?
At the root of the project directory run the following command:
        
    $ pytest -v

#### What are the relevant pivotal tracker stories?
#150834085
#150840244

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,7 @@ __pycache_/
 instance/
 .vscode/
 *.sqlite
+okbb_type.csv
+learningmap.csv
+skill.csv
+okbb.csv
```
Comment (LarryWachira) [inline .gitignore]: @andela-uibrahim Forgot to skip to a new line here.

File: test/test_manage.py [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
         os.environ["FLASK_CONFIG"] = "testing"
 
         initial_contributions_count = len(Contribution.query.all())
-        os.system("python manage.py seed")
+        os.system("python manage.py seed_test_data")
         new_contribution_count = len(Contribution.query.all())
 
         self.assertTrue(initial_contributions_count == 0)
-        self.assertGreater(new_contribution_count, initial_contributions_count)
+        self.assertGreater(new_contribution_count, initial_contributions_count)
```
Comment (LarryWachira) [inline test/test_manage.py]: No new line here too @andela-uibrahim ...

File: manage.py [hunk 3]
```diff
@@ -19,44 +25,146 @@
...
+    d1_dev_path_level_id = db.session.query(path_level).filter_by(path_id=developer_path.id, level_id=d1_level.id).first().path_level_id
+    d1_dev_path_skills = db.session.query(path_level_skill).filter_by(path_level_id=d1_dev_path_level_id).all()
+    skills = []
+    for path_id, skill_id in d1_dev_path_skills:    
+        skills.append(Skill.query.get(skill_id))
+        
+    print(developer_path_levels, "\n\n", d1_level_paths, "\n\n", d2_level_paths, "\n\n",  len(skills))
+
+
+
```
Comment (LarryWachira) [inline manage.py]: Should only skip two line here.

File: .gitignore (hunk 1 above)
Comment (andela-uibrahim) [reply .gitignore]: @LarryWachira changes have been implemented

File: migrations/versions/5226d6291e77_.py [hunk 4]
```diff
@@ -0,0 +1,226 @@
+"""empty message
```
Comment (LarryWachira) [inline migrations/versions/5226d6291e77_.py]: Seems like you added migrations, was this intentional?

**Comment by andela-milesanmi** [comment]
LGTM

**Comment by Mnickii** [comment]
LGTM

**Comment by wanjiku-carol** [comment]
LGTM

**Comment by danielwangai** [comment]
LGTM

**Pull request merged**